include/uv/win.h

```diff
@@ -606,7 +606,7 @@ typedef struct {
   struct uv_process_exit_s {                                                  \
     UV_REQ_FIELDS                                                             \
   } exit_req;                                                                 \
-  BYTE* child_stdio_buffer;                                                   \
+  void* unusued; /* TODO: retained for ABI compat; remove this in v2.x. */    \
```

Comment:
```suggestion
  void* unused; /* TODO: retained for ABI compat; remove this in v2.x. */    \
```
Or was that an intentional typo? :-)

Reply:
Yeah, I love inventing new words.